src/Adapter/MSTest.TestAdapter/Helpers/DataSerializationHelper.cs

```diff
@@ -13,6 +18,10 @@ internal static class DataSerializationHelper
         UseSimpleDictionaryFormat = true,
         EmitTypeInformation = System.Runtime.Serialization.EmitTypeInformation.Always,
         DateTimeFormat = new System.Runtime.Serialization.DateTimeFormat("O", CultureInfo.InvariantCulture),
+#if NETFRAMEWORK
+        DataContractSurrogate = SerializationSurrogateProvider.Instance,
```

Comment:
are DateOnly/TimeOnly structs only available on .NET core? or can then be used in netfx code via some nuget package, which is why we're adding the netfx code?